docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -262,6 +267,52 @@ docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 
 NOTE: Interested by tiny Docker images, check the {quarkus-images-url}/graalvm-{graalvm-version}/distroless[distroless] version.
 
+== Creating a container with a multi-stage Docker build
+
+The previous section showed you how to build a native executable using Maven, but implicitly required that the proper GraalVM version be installed on the building machine (be it your local machine or your CI/CD infrastructure).
+
+In cases where the GraalVM requirement cannot be met, you can use Docker to make the Maven build by using a multi-stage Docker build. A multi-stage Docker build is like two Dockerfile files combined in one, the first is used to build the artifact used by the second. 
```

Comment:
One last thing on my part:

`make the Maven build` -> `perform the Maven build`

Reply:
done